src/gen9_avc_encoder.c

```diff
@@ -4493,14 +4500,14 @@ gen9_avc_set_curbe_sfd(VADriverContextP ctx,
     memset(cmd,0,sizeof(gen9_avc_sfd_curbe_data));
 
     cmd->dw0.enable_intra_cost_scaling_for_static_frame = 1 ;
```

Comment:
The VDENC comment had better be removed in previous patch.
Otherwise it is changed back and forth.